aws/releases.go

```diff
@@ -49,12 +53,24 @@ var linux2Dist = &claircore.Distribution{
 	CPE:        cpe.MustUnbind("cpe:2.3:o:amazon:amazon_linux:2"),
 }
 
+var AL2023Dist = &claircore.Distribution{
+	Name:       "Amazon Linux",
+	DID:        ID,
+	Version:    "2023",
+	VersionID:  "2023",
+	PrettyName: "Amazon Linux 2023",
+	CPE:        cpe.MustUnbind("cpe:2.3:o:amazon:amazon_linux:2023"),
+}
+
+
 func releaseToDist(release Release) *claircore.Distribution {
 	switch release {
-	case Linux1:
-		return linux1Dist
-	case Linux2:
-		return linux2Dist
+	case AmazonLinux1:
+		return AL1Dist
+	case AmazonLinux2:
+		return AL2Dist
+	case AmazonLinux2023:
+		return AL2Dist
```

Comment:
This is incorrect, I think there is an intrinsic problem in the tests as `releaseToDist()` is called on the `got` and `want` values (via `scanner.parse`), meaning they are both equally wrong.

Reply:
Looks like I'll have to spend a moment understanding what's actually going on there rather than mostly just going through and adding/replacing the AL1/AL2 bits :)

Reply:
I just mean, the third branch here appears to erroneously returning `AL2Dist`, and given that it's disconcerting that we don't have a test to catch that.

Reply:
Oh, I guess I shouldn't look at reviews late at night. Thanks for the catch there, I'll see what I can figure out.